src/postgres.rs

```diff
@@ -1,16 +1,16 @@
-extern crate byteorder;
-extern crate num;
-
-use self::byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
-use self::num::Zero;
-use super::Decimal;
-use pg_crate::types::*;
-use std::error;
-use std::fmt;
-use std::io::Cursor;
-use std::result::*;
-use decimal::{div_by_u32, is_all_zero};
-use decimal::mul_by_u32;
+use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
+
+use num::Zero;
+
+use crate::Decimal;
```

Comment:
TODO: I suspect it's breaking these apart (similar in another file above) due to ordering issues. We should probably group crate vs std vs external.